scope/utils.py

```diff
@@ -102,6 +102,60 @@ def plot_light_curve_data(
         plt.savefig(save)
 
 
+def plot_periods(
+    features: pd.DataFrame,
+    limits: Optional[list] = None,
+    loglimits: Optional[bool] = False,
+    number_of_bins: Optional[int] = 20,
+    title: Optional[str] = None,
+    save: Optional[str] = None,
+):
+    """Plot a histogram of periods for the sample
+
+    """
+    # plot the H-R diagram for 1 M stars within 200 pc from the Sun
+    plt.rc("text", usetex=True)
+
+    # make figure
+    fig, ax = plt.subplots(figsize=(6, 6))
```

Comment:
Make it (16, 5) to be on par with the other plots?

Reply:
That's only true of lightcurves. gaia is (6,6) for example. Want me to still do 16, 5?

Reply:
Whatever is more pleasing to your experienced eye (mine are not).

Reply:
I think smaller is OK for the histograms here (we don't have much data).